onmetal-controller-manager/controllers/networking/loadbalancer_controller.go

```diff
@@ -115,7 +123,15 @@ func (r *LoadBalancerReconciler) findDestinations(ctx context.Context, log logr.
 	return destinations, nil
 }
 
-func (r *LoadBalancerReconciler) applyRouting(ctx context.Context, loadBalancer *networkingv1alpha1.LoadBalancer, destinations []commonv1alpha1.LocalUIDReference) error {
+func (r *LoadBalancerReconciler) findNetwork(ctx context.Context, log logr.Logger, loadBalancer *networkingv1alpha1.LoadBalancer) (*networkingv1alpha1.Network, error) {
```

Comment:
Can we call this method `getNetwork` because that is what essentially is happening here.

Reply:
done